src/footprintmonitoring/azext_footprintmonitoring/generated/custom.py

```diff
@@ -0,0 +1,267 @@
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+#
+# Code generated by Microsoft (R) AutoRest Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is
+# regenerated.
+# --------------------------------------------------------------------------
+# pylint: disable=line-too-long
+# pylint: disable=too-many-lines
+
+
+def footprintmonitoring_profile_list(client,
+                                     resource_group_name=None):
+    if resource_group_name:
+        return client.list_by_resource_group(resource_group_name=resource_group_name)
+    return client.list_by_subscription()
+
+
+def footprintmonitoring_profile_show(client,
+                                     resource_group_name,
+                                     profile_name):
+    return client.get(resource_group_name=resource_group_name,
+                      profile_name=profile_name)
+
+
+def footprintmonitoring_profile_create(client,
+                                       resource_group_name,
+                                       profile_name,
+                                       start_delay_milliseconds,
+                                       measurement_count,
+                                       tags=None,
+                                       location=None,
+                                       provisioning_state=None,
```

Comment:
I think this one should be marked as read_only in the swagger spec

Reply:
done, see above.